project/Dependencies.scala

```diff
@@ -44,6 +46,8 @@ object Dependencies {
     "com.okta.jwt" % "okta-jwt-verifier" % oktaJwtVerifierVersion,
     "com.okta.jwt" % "okta-jwt-verifier-impl" % oktaJwtVerifierVersion,
   )
+  val unirest = "com.konghq" % "unirest-java" % "4.0.0-RC2" % Test
+  val okHttp = "com.squareup.okhttp3" % "okhttp" % "5.0.0-alpha.10"
```

Comment:
Do we need to use an alpha version of okhttp?

Reply:
It's to resolve a dependency conflict.
It's a transitive dependency and I think sbt might be already using that version.
Will double check.

Reply:
Ok, explicit dependency (override) wasn't needed, just an exclusion.